cloud_experts_tutorials/rosa-mobb-azure-group-claims.adoc

```diff
@@ -0,0 +1,149 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-azure-group-claims"]
+= Tutorial: Configuring ROSA to use Azure AD group claims
```

Comment:
Tutorial: Configuring ROSA to use Microsoft Entra ID (formerly Azure Active Directory) group claims